index.js

```diff
@@ -16,8 +16,8 @@ var cookies = new Cookies()
 follow(process.argv[2], start)
 
 function setCookie (url, headers) {
-  var cookieHeader = headers['set-cookie']
-  if (!cookieHeader) return 0
+  if (!headers['set-cookie']) return 0
+  var cookieHeader = [].concat(headers['set-cookie'])
```

Comment:
Yeah, apparently not quite – it's a string if there's only one cookie, I believe. I've been using `spdy` for this before, but couldn't figure out how to get the actually used protocol and version out of it (`spdy` sadly doesn't set `res.httpVersion*`). But now that I'm thinking about this, it might be a good idea to fork `spdy` and/or `http2`and put in the missing parts.